### PR TITLE
Add bigWig to Wig converter

### LINKED ISSUE #12688: Requested converter: bigwig_to_wig
Recently a user requested a tool for performing the conversion from bigwig to wig.

### PR DESCRIPTION
**Pull request by gallardoalba**
This PR includes a bigwig to wig converter.

## How to test the changes?
(Select all options that apply)
- [x] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

Closes https://github.com/galaxyproject/galaxy/issues/12688

### CODE REVIEW

**Comment by bgruening** [comment]
> OSError: Error reading file '/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/datatypes/converters/bigwig_to_big_converter.xml': failed to load external entity "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/datatypes/converters/bigwig_to_big_converter.xml"

**Review by bgruening — COMMENTED** [review]
Looks good to me. Thanks @gallardoalba!

@nsoranzo do you recall what the current policy is with test-data? 
It's only 10kb so I think that is ok, but should we add it here or in the external repo?

File: lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">ucsc-bigwigtowig</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        bigWigToWig '${bigwig_input}'
+        #if $chrom
+            -chrom='${chrom}'
+        #end if
+        #if $start
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml]:
```suggestion
        #if str($start)
```

File: lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            -start=$start
+        #end if
+        #if $end
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml]:
```suggestion
        #if str($end)
```

File: lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Usage:
+   bigWigToWig in.bigWig out.wig
+
+Options:
+   - chrom=chr1 - if set restrict output to given chromosome
+   - start=N - if set, restrict output to only that over start
+   - end=N - if set, restict output to only that under end
+   - udcDir=/dir/to/cache - place to put cache for remote bigBed/bigWigs
+
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml]:
```suggestion
```

**Comment by nsoranzo** [comment]
> @nsoranzo do you recall what the current policy is with test-data? It's only 10kb so I think that is ok, but should we add it here or in the external repo?

I think it should be added to https://github.com/galaxyproject/galaxy-test-data

File: lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+            -start=$start
+        #end if
+        #if str($end)
+            -end=$end
+        #end if
+        output.wig
+    ]]></command>
+    <inputs>
+        <param name="bigwig_input" type="data" format="bigwig" label="BigWig input"/>
+        <param argument="-chrom" type="text" value="" optional="true" label="Restrict output to given chromosome" help="If set restrict output to given chromosome">
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml]:
```suggestion
        <param argument="-chrom" type="text" value="" optional="true" label="Restrict output to given chromosome">
```

File: lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+                    <add value="+" />
+                    <add value="-" />
+                    <add value=" " />
+                    <add value="_" />
+                </valid>
+            </sanitizer>
+            <validator type="regex">[A-Za-z0-9 -_+]+</validator>
+        </param>
+        <param argument="-start" type="integer" min="0" optional="true" label="Restrict output to only that over start" help="If set, restrict output to only that over start" />
+        <param argument="-end" type="integer" min="0" optional="true" label="Restict output to only that under end" help="If set, restict output to only that under end" />
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/bigwig_to_wig_converter.xml]:
```suggestion
        <param argument="-start" type="integer" min="0" optional="true" label="Restrict output to only that over start" />
        <param argument="-end" type="integer" min="0" optional="true" label="Restrict output to only that under end" />
```

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**